dogfooding/src/main/kotlin/io/getstream/video/android/ui/call/CallActivity.kt

```diff
@@ -68,36 +58,33 @@ class CallActivity : ComponentActivity() {
             }
         }
 
-        // step 3 - build a call screen
+        // step 3 - create the call UI
         setContent {
-            CallScreen(
+            CallContentScreen(
                 call = call,
-                showDebugOptions = io.getstream.video.android.BuildConfig.DEBUG,
                 onLeaveCall = {
                     call.leave()
                     finish()
-                }
-            )
```

Comment:
I'm not sure if this is a good approach because we should cover more use cases for joining a specific call (you can imagine Discord or Clubhouse, Twitter Live, or something) instead of ringing calls. We need to discuss more about this on the channel. So even if we may need to have some duplicate codes for Control Actions or something, I'd prefer separating them and making their role clear.